src/operator/mergeMapTo.ts

```diff
@@ -112,7 +112,7 @@ export class MergeMapToSubscriber<T, I, R> extends OuterSubscriber<T, I> {
     }
   }
 
-  private _innerSub(ish: any,
+  private _innerSub(ish: SubscribableOrPromise<I>,
```

Comment:
This is technically `ObservableInput<I>` because you could pass in an array or iterable.